src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -18,17 +18,15 @@
 from os import listdir
 from os.path import abspath, basename, exists, getsize, join, splitext
 
-import yaml
 from opera.pge.base.base_pge import PgeExecutor
 from opera.pge.base.base_pge import PostProcessorMixin
 from opera.pge.base.base_pge import PreProcessorMixin
 from opera.util.dataset_utils import parse_bounding_polygon_from_wkt
 from opera.util.error_codes import ErrorCode
 from opera.util.h5_utils import get_cslc_s1_product_metadata
 from opera.util.h5_utils import get_disp_s1_product_metadata
-from opera.util.h5_utils import MEASURED_PARAMETER_PATH_SEPARATOR
 from opera.util.input_validation import validate_algorithm_parameters_config, validate_disp_inputs
-from opera.util.render_jinja2 import render_jinja2
+from opera.util.render_jinja2 import augment_hd5_measured_parameters, render_jinja2
```

Comment:
I just realized I made a typo in the ticket description, I had intended this function to be named `augment_hdf5_measured_parameters`, sorry about that!

Reply:
I flipped a coin as to whether that was a typo and it came up wrong I guess :)

Reply:
I wouldn't bet against my own stupidity if I were you! 😆 😢